layers/layertypes.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 var (
+	// Pending to set the proper number
+	LayerTypeTLS                          = gopacket.RegisterLayerType(1000, gopacket.LayerTypeMetadata{Name: "TLS", Decoder: gopacket.DecodeFunc(decodeTLS)})
```

Comment:
Just throw this at the bottom of the list, incrementing the top-most current number by one :)